assets/dev/js/editor/components/ai/components/prompt-search.js

```diff
@@ -0,0 +1,15 @@
+import { forwardRef } from 'react';
+import SearchField from './ui/search-field';
+
+const PromptSearch = forwardRef( ( props, ref ) => {
+	return (
+		<SearchField
+			name="prompt"
+			placeholder={ __( 'Describe the text and tone you want to use...', 'elementor' ) }
```

Comment:
not sure if the "..." should be inside the translation (also others)

Reply:
Done.